specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -5685,6 +5877,157 @@
         }
       },
       "description": "The list usages operation response."
+    },
+    "CanMigrateInput": {
+      "type": "object",
+      "description": "Input for canMigrate operation.",
+      "properties": {
```

Comment:
for classic AFD/CDN to AFDX, is it portal's work to provide the copied/existing WAF or will RP generate WAF automatically? From my understanding, portal will provide the WAF. If so, should we pass the WAF reference to CanMigrateInput?